src/main/java/org/broadinstitute/dsm/jobs/UPSTrackingJob.java

```diff
@@ -32,15 +30,17 @@
 public class UPSTrackingJob implements Job {
 
     private static final Logger logger = LoggerFactory.getLogger(UPSTrackingJob.class);
-    private static final String SQL_SELECT_KITS = "SELECT * FROM ddp_kit kit LEFT JOIN ddp_kit_request req " +
-            "ON (kit.dsm_kit_request_id = req.dsm_kit_request_id) WHERE req.ddp_instance_id = ? and kit_label not like \"%\\_1\" order by kit.dsm_kit_request_id ASC";
+            private static final String SQL_SELECT_KITS = "SELECT * FROM ddp_kit kit LEFT JOIN ddp_kit_request req " +
+                    " ON (kit.dsm_kit_request_id = req.dsm_kit_request_id) WHERE req.ddp_instance_id = 9 and kit_label not like \"%\\\\_1\"  and (ups_tracking_status not like \"D %\" or ups_return_status not like \"D %\")" +
```

Comment:
I'm a little concerned about the magic `D %` expression here.  Can you apply `UPSStatus.DELIVERED_TYPE` here?